content/en/tutorials/cloud-integration-testing-localStack-ci/index.md

```diff
@@ -0,0 +1,282 @@
+
+## Prerequisites
+
+1. LocalStack Pro with [LocalStack Auth Token](https://docs.localstack.cloud/getting-started/auth-token/)
+2. [Docker](https://docs.docker.com/engine/install/)
+3. [Python-3](https://www.python.org/downloads/)
+4. [LocalStack web application account](https://app.localstack.cloud/sign-in)
+5. [GitHub account](https://github.com/) & [`gh` CLI](https://github.com/cli/cli?tab=readme-ov-file#installation) (optional)
+
+## Architecture Overview
+
+We will be using the demo - [Serverless image resizer](https://github.com/localstack-samples/sample-serverless-image-resizer-s3-lambda/tree/main) for our tutorial. In this serverless image resizer application, a simple frontend  application facilitates image uploads. A separate lambda generates pre-signed URLs for direct upload to a designated S3 bucket. File uploads to this bucket triggers a python lambda that handles image resizing. Another lambda takes charge of listing all uploaded and resized images, additionally generating pre-signed URLs specifically for the browser to display them. 
+
+The architecture incorporates error handling by routing lambda failures to a 'Simple Notification Service' (SNS) topic, which in turn sends an email notification, through 'Amazon Simple Email Service'(SES). 
+
+Using the LocalStack internal `/_localstack/aws/ses` endpoint, we can run end-to-end integration tests to verify that emails have been sent correctly.
+
+![Architecture Diagram](./system_overview.png "Architecture Diagram")
+
+
+## Create the GitHub Action Workflow
+
+We will create a GitHub Action workflow called "Run Integration Tests" to automate the integration testing process for our demo sample. This workflow will execute various steps to ensure smooth testing and monitoring of our application's functionality. It will be covering key aspects such as:
+- Setting up Python 3+, docker, installing project dependencies, starting LocalStack for cloud emulation
+-  Deploying Infrastructure 
+-  Running tests with `pytest` 
+-  Enabling localStack logs, sending Slack notifications on failure, and generating & uploading diagnostic reports for troubleshooting purposes.
+  
+  
+To begin, let's clone the [repository](https://github.com/localstack-samples/sample-serverless-image-resizer-s3-lambda/tree/main) and fork it. Afterward, we'll proceed with the following steps:
+
+- Create a new directory named `.github` in the root of the repository.
+- Inside the `.github` directory, create a sub-directory named `workflows`.
+- Within the `workflows` sub-directory, create a new file called `integration-test.yml`.
+
+Next, we'll outline the various steps needed to build our integration testing workflow. Let's get started!
+
+### Setting Up Development Environment with LocalStack
+
+We will be setting up dependencies and starting localstack in the github runner.
+
+**Step 1: Checkout & set up Python 3+**
```

Comment:
I would avoid using bold steps to highlight the walkthrough. Please stick to headings and sub-headings as much as possible